build/pack.c

```diff
@@ -14,6 +14,8 @@
 #include <rpm/rpmfileutil.h>
 #include <rpm/rpmlog.h>
 
+#include <prtpool.h>			/* NSPR thread pools */
```

Comment:
Right, openssl support is very recent, so I missed that. Do you have suggestions on what could provide a good thread pool API, if not nspr? I wouldn't want to do a poor internal re-implementation on top of pthreads.

The whole thing can also be made conditional.